package.json

```diff
@@ -26,7 +26,7 @@
   "scripts": {
     "build": "tsc --build && yarn copy-files",
     "clean": "tsc --build --clean",
-    "copy-files": "copyfiles -u 1 src/**/*.scss dist/",
+    "copy-files": "copyfiles -u 1 src/**/*.module.scss dist/",
```

Comment:
`*.module.scss` will still be matched against `*.scss` so this isn't necessary :)
It would actually be safer to leave it in case there are internal scss files (eg. mixins consumed by a file with .module.scss) which are valid.